Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneLifeCycleState.cs

```diff
@@ -0,0 +1,17 @@
+using Arch.Core;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ECS.SceneLifeCycle
+{
+    public class SceneLifeCycleState
```

Comment:
Tidy up this class:  clearly indicate what is read-only and what is not, what can be modified my consumers and what can't be